src/decoder.rs

```diff
@@ -56,17 +56,19 @@ fn decode_bundle(msg: &[u8]) -> Result<OscPacket> {
 
     let mut bundle: Vec<OscPacket> = Vec::new();
 
-    let mut elem_size = read_bundle_element_size(&mut cursor)?;
+    if msg.len() > cursor.position() as usize {
```

Comment:
Can you replace this with an early return?

```rust
if msg.len() <= cursor.position() as usize { return }
```

I would like keep the unchanged lines from being part of the diff.

Reply:
I've amended the commit with an early return